src/internal/packager/sbom/catalog.go

```diff
@@ -106,6 +106,12 @@ func Catalog(componentSBOMs map[string]*types.ComponentSBOM, tagToImage map[name
 		currImage++
 	}
 
+	if len(componentSBOMs) != 0 && len(tagToImage) != 0 {
```

Comment:
nit, style-wise we usually have used `len() < 1` vs `len() != 0`.

Reply:
Changed to len() > 0